.mailmap

```diff
@@ -654,6 +654,8 @@ Vitali Haravy <HumaneProgrammer@gmail.com> Vitali Haravy <humaneprogrammer@gmail
 Vitaly Shukela <vi0oss@gmail.com>
 Waffle Lapkin <waffle.lapkin@gmail.com>
 Waffle Lapkin <waffle.lapkin@gmail.com> <waffle.lapkin@tasking.com>
+Weihang Lo <me@weihanglo.tw>
+Weihang Lo <me@weihanglo.tw> <weihanglo@users.noreply.github.com>
```

Comment:
(unrelated: why I saw some people have a random number prefix `18237433+user@users.noreply.github.com`, but not for mine?)

Reply:
See https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#about-commit-email-addresses

![Screenshot 2025-01-02 231612](https://github.com/user-attachments/assets/1921c128-672c-4bc7-82e1-c840a616e2b3)

Reply:
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExb2N1eDVubW41OHhnenQxMG1ydzBpYnZuamM5a3J1YjZhb2tweGJyMyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/kGiKJNlIljFdinlJsB/giphy.webp)

Reply:
TL;DR: your account is OG